bubblesub/api/threading.py

```diff
@@ -66,7 +66,9 @@ def __init__(self, log_api: LogApi) -> None:
         self._log_api = log_api
         self._running = False
         self._clearing = False
-        self._queue: queue.Queue[T.Any] = queue.Queue()
+        self._queue: queue.Queue[  # pylint: disable=E1136
```

Comment:
I used this id because the string was too long. I've also seen that in another parts of the code you have used error numbers

Reply:
What about this one?

Reply:
If there is a way I'd really prefer to have string identifiers, and if I used numbers elsewhere then I'm either a fool, or I was forced to and sad

Reply:
I think you were forced to because the strings were too long. I could try to put them on a different line in case they are too long, but I don't assure anything